### ISSUE TITLE
admin search

### ISSUE THREAD

**Issue by infosws** [woocommerce/woocommerce#19383]
## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [ ] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [ ] I have attempted to find the simplest possible steps to reproduce the issue
- [ ] I have included a failing test as a pull request (Optional)

## Steps to reproduce the issue

1.
2.
3.

## Expected/actual behavior

When I follow those steps, I see...

I was expecting to see...

## Isolating the problem

- [ ] This bug happens with only WooCommerce plugin active
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [ ] I can reproduce this bug consistently using the steps above

## WordPress Environment

<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin here.
```
</details>

**Comment by mikejolley** [comment]
How is anyone supposed to diagnose an issue when the only detail is "admin search"?

**Comment by infosws** [comment]
@mikejolley it posted before I wanted it too, some of the details are there now, just checking the rest to make sure i've covered the list.

Thank you.

**Comment by infosws** [comment]
Hi @mikejolley,

I now have a complete list there for you, hopefully this is all the information you will need. 
If anything else is needed, please let me know.

Thanks

**Comment by GlennMartin1** [comment]
Your issue will likely be resolved with https://github.com/woocommerce/woocommerce/pull/19096 which is still a work-in-progress.

**Comment by infosws** [comment]
Hi @GlennMartin1,

With the posts I did look over, I see that some other issues are affecting the search currently but I didn't see something like this exsactly hence listing it.

I'll watch #19096 and see how it goes, is a fix planned for 3.3.4 or a later version?

Thanks

**Comment by GlennMartin1** [comment]
Set for WC 3.4.0, unless you know how to test the pull request.
